packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -243,6 +247,16 @@ class ExpansionPanelList extends StatefulWidget {
   /// is null, then [ThemeData.dividerColor] is used.
   final Color? dividerColor;
 
+  /// The color of the arrow icon when the panel is collapsed.
+  ///
+  /// {@macro flutter.material.ExpandIcon.color}
```

Comment:
Looks like there is no template defined for this macro? That's why the doc check is failing.